packages/jest-util/src/install_common_globals.js

```diff
@@ -19,7 +19,7 @@ export default function(globalObject: Global, globals: ConfigGlobals) {
   globalObject.process = createProcesObject();
 
   // Keep a reference to "Promise", since "jasmine_light.js" needs it.
-  global[global.Symbol.for('jest-native-promise')] = Promise;
+  globalObject[globalObject.Symbol.for('jest-native-promise')] = Promise;
```

Comment:
`globalObject.Symbol` is not defined according to CI

Reply:
Yeah, the test was wrong. I updated it :)